typescript/cli/src/deploy/utils.ts

```diff
@@ -61,3 +68,13 @@ export async function runPreflightChecksForChains({
   await assertGasBalances(multiProvider, signer, chains, minGas);
   logGreen('Balances are sufficient ✅');
 }
+
+export function isAdvancedISMConfig(
+  config: ChainMap<MultisigConfig> | ChainMap<IsmConfig>,
+): boolean {
+  return Object.values(config).some((c) => 'type' in c);
+}
+
+export function isAdvancedZODISMConfig(filepath: string): boolean {
```

Comment:
Sorry this name is confusing me a bit. When is something a zod config?

Reply:
`isAdvancedISMConfig` is for checking in core once we have parsed the config as `ChainMap<Multisig> | ChainMap<IsmConfig> `and `isAdvancedZODISMConfig` is directly checking if the yaml provided satisfies the `ISMConfigMap.safeParse()`

Reply:
agree this is confusing